web/app/js/components/Navigation.jsx

```diff
@@ -131,8 +136,10 @@ class NavigationBase extends React.Component {
     super(props);
     this.api = this.props.api;
     this.handleApiError = this.handleApiError.bind(this);
+    this.handleCommunityClick = this.handleCommunityClick.bind(this);
 
     this.state = this.getInitialState();
+    this.state.hideUpdateBadge = true;
```

Comment:
should setting `hideUpdateBadge` just be part of `getInitialState`?

Reply:
good point!